.github/workflows/generate-all.yml

```diff
@@ -0,0 +1,103 @@
+name: generate-all
+
+on:
+  schedule:
+    - cron: '15 */4 * * *'
+  push:
+    branches:
+      - 'develop'
+      - 'feature/**'
+      - 'chore/**'
+      - 'bugfix/**'
+      - 'story//**'
+  workflow_dispatch:
+
+jobs:
+  prerequisite:
+    name: Prerequisite for Generation
+    if: github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/heads/feature')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Prerequisite
+        run: |
+          echo Prerequisites passed!
+
+  generate-bmcapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/bmc/download_spec
+      package-name: pnap_bmc_api
+  generate-ranchersolutionapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/rancher/download_spec
+      package-name: pnap_rancher_solution_api
+  generate-networkapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/networks/download_spec
+      package-name: pnap_network_api
+  generate-ipapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/ips/download_spec
+      package-name: pnap_ip_api
+  generate-auditapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/bmc-audit/download_spec
+      package-name: pnap_audit_api
+  generate-tagapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/tags/download_spec
+      package-name: pnap_tag_api
+  generate-networkstorageapi:
+    needs: prerequisite
+    uses: ./.github/workflows/_generate.yml
+    with:
+      spec-link: https://developers.phoenixnap.com/portals/api/sites/phoenixnap-prod-developerportal/liveportal/apis/network-storage/download_spec
+      package-name: pnap_network_storage_api
+
+  # Push generated code
+  push-changes:
+    name: Push Changes
+    runs-on: ubuntu-latest
+    needs:
+      - generate-bmcapi
+      - generate-ranchersolutionapi
+      - generate-networkapi
+      - generate-ipapi
+      - generate-auditapi
+      - generate-tagapi
+      - generate-networkstorageapi
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+        # with:
```

Comment:
Can we remove this commented section?

Reply:
I think it should already be removed now...